src/burnchains/burnchain.rs

```diff
@@ -394,38 +394,61 @@ impl Burnchain {
     pub fn get_db_path(&self) -> String {
         let chainstate_dir = Burnchain::get_chainstate_path(&self.working_dir, &self.chain_name, &self.network_name);
         let mut db_pathbuf = PathBuf::from(&chainstate_dir);
-        db_pathbuf.push("burn.db");
+        db_pathbuf.push("sortition.db");
+        
+        let db_path = db_pathbuf.to_str().unwrap().to_string();
+        db_path
+    }
+
+    pub fn get_burnchaindb_path(&self) -> String {
+        let chainstate_dir = Burnchain::get_chainstate_path(&self.working_dir, &self.chain_name, &self.network_name);
+        let mut db_pathbuf = PathBuf::from(&chainstate_dir);
+        db_pathbuf.push("burnchain.db");
         
         let db_path = db_pathbuf.to_str().unwrap().to_string();
         db_path
     }
 
-    fn connect_db<I: BurnchainIndexer>(&self, indexer: &I, readwrite: bool) -> Result<BurnDB, burnchain_error> {
+    fn connect_db<I: BurnchainIndexer>(&self, indexer: &I, readwrite: bool) -> Result<(SortitionDB, BurnchainDb), burnchain_error> {
```

Comment:
`(SortitionDB, BurnchainDb)`

My eye involuntarily twitched reading that

Reply:
Ha ha, same here